Source/WebCore/page/ResourceUsageThread.h

```diff
@@ -35,6 +35,7 @@
 #include <wtf/Lock.h>
 #include <wtf/NeverDestroyed.h>
 #include <wtf/Noncopyable.h>
+#include <WebCore/MemoryCache.h>
```

Comment:
This needs to be `#include "MemoryCache"`

Reply:
done

Reply:
Hi @mcatanzaro,
 Also, I am looking Layers Memory TimeLine on web inspector...
we have to calculate all the Graphics Layer memory for a layer.
so, may I get the memory from the `double RenderLayerBacking::backingStoreMemoryEstimate()` or `CompositingCoordinator` or something else..
can you please suggest me?

Reply:
Sorry, I don't know.